qiskit_experiments/library/randomized_benchmarking/mirror_rb_analysis.py

```diff
@@ -0,0 +1,674 @@
+            series=curve_data.data_allocation,
+            xdata=curve_data.x,
+            ydata=curve_data.y,
+            sigma=curve_data.y_err,
+            shots=curve_data.shots,
+            method="sample",
+        )
+
+        # sort by x value in ascending order
+        data_allocation, xdata, ydata, sigma, shots = curve.utils.data_sort(
+            series=data_allocation,
+            xdata=xdata,
+            ydata=ydata,
+            sigma=sigma,
+            shots=shots,
+        )
+
+        return curve.CurveData(
+            x=xdata,
+            y=ydata,
+            y_err=sigma,
+            shots=shots,
+            data_allocation=data_allocation,
+            labels=curve_data.labels,
+        )
+
+    def _create_analysis_results(
+        self,
+        fit_data: curve.FitData,
+        quality: str,
+        **metadata,
+    ) -> List[AnalysisResultData]:
+        """Create analysis results for important fit parameters.
+
+        Args:
+            fit_data: Fit outcome.
+            quality: Quality of fit outcome.
+
+        Returns:
+            List of analysis result data.
+        """
+
+        outcomes = super()._create_analysis_results(fit_data, quality, **metadata)
+        num_qubits = len(self._physical_qubits)
+
+        # nrb is calculated for both EPC and EI per the equations in the docstring
+        ei_nrb = 4**self._num_qubits
+        ei_scale = (ei_nrb - 1) / ei_nrb
+        epc_nrb = 2**self._num_qubits
+        epc_scale = (epc_nrb - 1) / epc_nrb
+
+        alpha = fit_data.ufloat_params["alpha"]
+
+        # Calculate EPC and EI per the equations in the docstring
+        epc = epc_scale * (1 - alpha)
+        ei = ei_scale * (1 - alpha)
+
+        outcomes.append(
+            AnalysisResultData(
+                name="EPC", value=epc, chisq=fit_data.reduced_chisq, quality=quality, extra=metadata
+            )
+        )
+        outcomes.append(
+            AnalysisResultData(
+                name="EI", value=ei, chisq=fit_data.reduced_chisq, quality=quality, extra=metadata
+            )
+        )
+
+        # Correction for 1Q depolarizing channel if EPGs are provided
+        if self.options.epg_1_qubit and num_qubits == 2:
+            epc = _exclude_1q_error(
+                epc=epc,
+                qubits=self._physical_qubits,
+                gate_counts_per_clifford=self._gate_counts_per_clifford,
+                extra_analyses=self.options.epg_1_qubit,
+            )
+            outcomes.append(
+                AnalysisResultData(
+                    name="EPC_corrected",
+                    value=epc,
+                    chisq=fit_data.reduced_chisq,
+                    quality=quality,
+                    extra=metadata,
+                )
+            )
+
+        # Calculate EPG
+        if self._gate_counts_per_clifford is not None and self.options.gate_error_ratio:
+            epg_dict = _calculate_epg(
```

Comment:
Can we apply the same logic to MRB alpha?

Reply:
I think this comment doesn't apply any more now that the analysis class is subclassing from `RBAnalysis`?